packages/sanity/src/core/studio/upsell/index.ts

```diff
@@ -1,2 +1,5 @@
 export * from './__telemetry__/'
+export * from './types'
 export * from './upsellDescriptionSerializer'
+export {UpsellDialog} from './UpsellDialog'
+export {UpsellPanel} from './UpsellPanel'
```

Comment:
Had to explicitly export in order to use it in the moved contexts into the scheduled-publishing folder